Lombiq.NodeJs.Extensions/ExecWithMutex.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.Build.Framework;
+using Microsoft.Build.Tasks;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Lombiq.NodeJs.Extensions;
+
+/// <summary>
+/// Like an Exec task, but with critical section.
+/// </summary>
+public class ExecWithMutex : Exec
+{
+    /// <summary>
+    /// Gets or sets mutex name.
+    /// </summary>
+    [Required]
+    public string MutexName { get; set; }
+
+    /// <inheritdoc />
+    public override bool Execute()
+    {
+        var timeout = TimeSpan.FromSeconds(60);
```

Comment:
Can we make this configurable?

Reply:
Where would you configure it?

Reply:
As a `msbuild` parameter, if I remember well `dotnet build /p:PARAM=VALUE` or something similar.

Reply:
Why would you want to configure it?

Reply:
Because this seems something where we can tune the build process if the 60 seconds is not enough in a bigger solution.

Reply:
While I'd argue that this is overkill right now, I do agree that making it configurable helps with discoverability. So, I'll add an MSBuild property for that which can be overridden when needed.